test/e2e/allocation.cancel.test.js

```diff
@@ -0,0 +1,26 @@
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationJourney } from './pages/'
+
+fixture('Cancel allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+  await allocationJourney.createAllocation()
+  await t.navigateTo(allocations)
+})
+test('Cancel allocation', async t => {
+  await t.click(
+    allocationJourney.allocationCancelPage.nodes.linkToFirstAllocation
+  )
+  await allocationJourney.scrollToBottom()
+  await t.click(allocationJourney.allocationCancelPage.nodes.cancelLink)
+  await t
+    .expect(allocationJourney.getCurrentUrl())
+    .match(/\/allocation\/[\w]{8}(-[\w]{4}){3}-[\w]{12}\/cancel\/reason$/)
+  await t.click(allocationJourney.allocationCancelPage.nodes.submitButton)
```

Comment:
Can we use `page.submitForm()` for consistency please. Then you don't need to create `allocationJourney.allocationCancelPage.nodes.submitButton` either.

Reply:
Let's leave this for a future improvement actually.